core/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -503,6 +509,15 @@ public QueryStringQueryBuilder fuzzyRewrite(String fuzzyRewrite) {
         return this;
     }
 
+    /** Gets the fuzzy query transposition setting. */
```

Comment:
This javadoc doesn't tell you any more than the method name does. I'd drop it or explain the setting in some detail. Also it doesn't really make sense to have javadoc on the get portion but not the set portion.